pallets/subtensor/src/epoch.rs

```diff
@@ -452,34 +480,58 @@ impl<T: Config> Pallet<T> {
         // =================================
 
         // Compute normalized emission scores. range: I32F32(0, 1)
-        let mut normalized_emission: Vec<I32F32> = incentive.iter().zip( dividends.clone() ).map( |(ii, di)| ii + di ).collect();
-        inplace_normalize( &mut normalized_emission );
+        let combined_emission: Vec<I32F32> = incentive.iter().zip( dividends.clone() ).map( |(ii, di)| ii + di ).collect();
+        let emission_sum: I32F32 = combined_emission.iter().sum();
+
+        let mut normalized_server_emission: Vec<I32F32> = incentive.clone(); // Servers get incentive.
+        let mut normalized_validator_emission: Vec<I32F32> = dividends.clone(); // Validators get dividends.
+        let mut normalized_combined_emission: Vec<I32F32> = combined_emission.clone();
+        // Normalize on the sum of incentive + dividends.
+        inplace_normalize_using_sum( &mut normalized_server_emission, emission_sum ); 
+        inplace_normalize_using_sum( &mut normalized_validator_emission, emission_sum );
+        inplace_normalize( &mut normalized_combined_emission );
 
         // If emission is zero, replace emission with normalized stake.
-        if is_zero( &normalized_emission ) { // no weights set | outdated weights | self_weights
+        if emission_sum == I32F32::from(0) { // no weights set | outdated weights | self_weights
             if is_zero( &active_stake ) { // no active stake
-                normalized_emission = stake.clone(); // do not mask inactive, assumes stake is normalized
+                normalized_server_emission = stake.clone(); // do not mask inactive, assumes stake is normalized
```

Comment:
Rather `normalized_validator_emission = stake.clone();` which will reward those delegating stake as well, and not just the hotkey owner.